src/client/webpack.config.js

```diff
@@ -145,8 +161,9 @@ if (isProductionMode){
     }
     return loader;
   });
+
   webpackConfig.plugins.push(
-    new ExtractTextPlugin('[name].[contenthash].css'),
+    new ExtractTextPlugin('[name].css?[contenthash]'),
```

Comment:
no but it follows the pattern for hash breaking that the .js files use as query arguments rather than part of the filename